exercises/diffie-hellman/diffie-hellman.spec.js

```diff
@@ -0,0 +1,67 @@
+import DiffieHellman from './diffie-hellman';
+
+describe('diffie-hellman', () => {
+  const p = 23;
+  const g = 5;
```

Comment:
I'd give better names to `p` and `g`, but that is the terms used in the `README`, so they're ok for me.